components/server/test/ome/server/itests/pixeldata/PersistentEventLogLoaderTest.java

```diff
@@ -273,9 +274,9 @@ protected Pixels pixels() {
         pixels.setSizeZ(1);
         pixels.setSizeT(1);
         pixels.setSizeC(1);
-        pixels.setSha1("UNKNOWN");
-        pixels.setPixelsType(new PixelsType("int8"));
-        pixels.setDimensionOrder(new DimensionOrder("XYZCT"));
+        pixels.setSha1(ChecksumAlgorithm.VALUE_ADLER_32);
```

Comment:
It is like "pending..." (and therefore needs to be cleaned up / rationalized) but I think changing it to another, unexpected value ("Adler-32") is probably out of scope of this PR.